### ISSUE TITLE
Use current Node LTS (or newer) in CI

### ISSUE THREAD

**Issue by AndrewADev** [openseadragon/openseadragon#2305]
I noticed that the CI is running [Node 16](https://github.com/openseadragon/openseadragon/blob/master/.travis.yml#L4), which has switched to maintenance since [the end of last year.](https://nodejs.dev/en/about/releases/)

It seems there was a [pull request](https://github.com/openseadragon/openseadragon/pull/2244) that reverted to v16 due to issues with the [tests not running](https://github.com/openseadragon/openseadragon/pull/2244#issuecomment-1329845829)

I just ran the tests (also coverage command) locally on both Node 16 and 18, and while there were errors, they looked like those described [here](https://github.com/openseadragon/openseadragon/issues/1680#issuecomment-802254095).

So, I was wondering if it might be worth revisiting updating the CI to at least current LTS Node or newer? (perhaps the previous issues were fixed with #2298 ?) 

Alternatively, what were the symptoms observed when node version was reverted in the CI?

**Comment by AndrewADev** [comment]
I took another look at the change, and it was from `stable` -> `16.14.2`. I don't see [a listing for `stable` explicitly in the Travis docs](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions), but it seems like it means the most recent, stable version, which is not necessarily the LTS. So, it seems the tests were probably failing due to Node 19 compatibility issues right after it became current stable.

**Comment by iangilman** [comment]
Sounds good! Can you make a PR with the change? Is it possible to change it to 'lts' or so, or do we need to explicitly list 18?

**Comment by AndrewADev** [comment]
> Sounds good! Can you make a PR with the change? Is it possible to change it to 'lts' or so, or do we need to explicitly list 18?

It is possible to target LTS, though it seems I also encountered an error on my first attempt at that (possibly the same one you did back when the initial change was made).

I was able to work around the error I encountered by requesting a newer version of the runner (was more related to the environment required for node, than the node environment not supporting the project/dependencies). I would think that can be removed later as well once Travis updates its default runner version. I've left some details in [a comment on the PR](https://github.com/openseadragon/openseadragon/pull/2308#issuecomment-1448851411).
